### PR TITLE
Ocrvs 2427 Added DECLARATION_UPDATED task

### PR DESCRIPTION
**Pull request by Zangetsu101**
This will ensure that changes made by registrationAgent/registrar before validating/registering a declaration will be tracked.

### CODE REVIEW

**Comment by euanmillar** [comment]
@Zangetsu101 can you attend to the conflicts in this PR please?

**Comment by Zangetsu101** [comment]
@euanmillar Actually this ticket is still incomplete. Now that the correction flow is merged I'll continue this ticket. Sorry for the confusion.

File: packages/workflow/src/features/registration/fhir/fhir-template.ts [hunk 1]
```diff
@@ -19,6 +19,12 @@ import { getEventType } from '@workflow/features/registration/utils'
 
 export const INFORMANT_CODE = 'informant-details'
 
+export function isTask(
```
Comment (tahmidrahman-dsi) [inline packages/workflow/src/features/registration/fhir/fhir-template.ts:22]: Nice one 👍🏼

**Review by tahmidrahman-dsi — APPROVED** [review]
LGTM!

**Pull request merged**